src/pages/DataAccessRequestApplication_new.js

```diff
@@ -892,13 +892,15 @@ class DataAccessRequestApplicationNew extends Component {
 
     const { dataRequestId } = this.props.match.params;
     const eRACommonsDestination = isNil(dataRequestId) ? 'dar_application' : ('dar_application/' + dataRequestId);
-    const { problemSavingRequest, showValidationMessages,  step1 } = this.state;
+    const { showValidationMessages,  step1 } = this.state;
     const isTypeOfResearchInvalid = this.isTypeOfResearchInvalid();
     const ontologies = this.formatOntologyItems(this.state.formData.ontologies);
-    const step1Invalid = this.step1InvalidResult(this.step1InvalidChecks());
-    const step2Invalid = this.verifyStep2();
-    const step3Invalid = this.step3InvalidResult();
-    const libraryCardInvalid = isEmpty(getOr([], 'libraryCards', this.state.researcher)) && !checkNihDataOnly;
+
+    // VALIDATION CHECKS:
+    // const step1Invalid = this.step1InvalidResult(this.step1InvalidChecks());
+    // const step2Invalid = this.verifyStep2();
+    // const step3Invalid = this.step3InvalidResult();
+    // const libraryCardInvalid = isEmpty(getOr([], 'libraryCards', this.state.researcher)) && !checkNihDataOnly;
```

Comment:
Are we waiting on all other steps to be merged before implementing these? If so, we should add that as a TODO comment here and make sure we have a follow-on story to apply the validation

Reply:
This is correct. I assumed that there would be a ticket for all of the final gluing together of the components, and the actual validation / submission work.